observability/monitoring/common-monitoring-configuration-scenarios.adoc

```diff
@@ -0,0 +1,41 @@
+
+After you complete the following configuration activities, users can view collected metrics, query these metrics, and receive alerts for their own projects:
+
+* xref:../../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects[Enable user-defined projects.]
+* xref:../../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#granting-users-permission-to-monitor-user-defined-projects_enabling-monitoring-for-user-defined-projects[Assign the `monitoring-rules-view`, `monitoring-rules-edit`, or `monitoring-edit` cluster roles] to grant non-administrator users permissions to monitor user-defined projects.
+* If needed, xref:../../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#granting-users-permission-to-configure-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects[assign the `user-workload-monitoring-config-edit` cluster role] to grant non-administrator users permission to configure user-defined projects.
+* xref:../../observability/monitoring/enabling-alert-routing-for-user-defined-projects.adoc#enabling-alert-routing-for-user-defined-projects[Enable alert routing for user-defined projects] so that developers and other users can configure custom alerts and alert routing for their projects.
+* If needed, configure alert routing for user-defined projects either to xref:../../observability/monitoring/enabling-alert-routing-for-user-defined-projects.adoc#enabling-the-platform-alertmanager-instance-for-user-defined-alert-routing_enabling-alert-routing-for-user-defined-projects[use the Alertmanager instance deployed for core platform monitoring] or to xref:../../observability/monitoring/enabling-alert-routing-for-user-defined-projects.adoc#enabling-a-separate-alertmanager-instance-for-user-defined-alert-routing_enabling-alert-routing-for-user-defined-projects[use an optional Alertmanager instance dedicated for use by user-defined projects].
+* If needed, xref:../../observability/monitoring/managing-alerts.adoc#configuring-different-alert-receivers-for-default-platform-alerts-and-user-defined-alerts_managing-alerts[configure different alert receivers for core platform alerts and alerts for user-defined projects.]
+* If needed, xref:../../observability/monitoring/managing-alerts.adoc#applying-a-custom-configuration-to-alertmanager-for-user-defined-alert-routing_managing-alerts[apply a custom configuration to Alertmanager for user-defined alert routing.]
+
+After you enable and configure monitoring for user-defined projects, developers and other users can then perform the following activities to set up and use monitoring for their own projects:
+
+* xref:../../observability/monitoring/managing-metrics.adoc#setting-up-metrics-collection-for-user-defined-projects_managing-metrics[Deploy and monitor services] in their project.
```

Comment:
Here we use `Project` whereas the following points use `Projects`.

Reply:
fixed